plugin/DataView/DataDetailView.php

```diff
@@ -97,6 +100,22 @@ class DataDetailView
 	/** @var int */
 	private $_movieLinks = MovieLinkTypes::MOVIE_LINKS_NONE;
 
+	/** @var bool */
+	private $_dataDetailViewActive = false;
+
+	/** @var DataViewSimilarEstates */
+	private $_pDataViewSimilarEstates = null;
```

Comment:
A comment would be good that states that the property `private $_pDataViewSimilarEstates` and its getter need to stay for compatibility reasons.